### PR TITLE
Small fixes.

### PR DESCRIPTION
**Pull request by TheLocalPub**
* Deleted the start_in_vehicle module
* Deleted the outdated setting.sqf file in set_face module
* Fixed some typos in the call examples of the marker_control module.

### CODE REVIEW

File: modules/marker_control/settings.sqf [hunk 1]
```diff
@@ -12,7 +12,7 @@
 // Adds all markers that are on the named layer
 
 // Example:
-// [blufor, "markerBlu", true] call FUNC(Add);
-// [[blufor, civilian], "markerBlu2", false] call FUNC(Add);
-// [opfor, "markerRed", true] call FUNC(Add);
+// [blufor, "markerBlu", true] call FUNC(Addlayer);
```
Comment (PiZZAD0X) [inline modules/marker_control/settings.sqf:15]: Not all calls need to be AddLayer: 

if you want to affect all markers on a layer: addLayer
if you want to affect only a named marker: add

**Comment by EM-Creations** [comment]
So how come start in vehicle is removed?

Also this should target the dev branch please.

**Comment by TheLocalPub** [comment]
> So how come start in vehicle is removed?
> 
> Also this should target the dev branch please.

I looked at the module a few days ago and noticed no settings file for it so I asked pizza and he said it was a redundant module.

**Comment by EM-Creations** [comment]
> > So how come start in vehicle is removed?
> > Also this should target the dev branch please.
> 
> I looked at the module a few days ago and noticed no settings file for it so I asked pizza and he said it was a redundant module.

OK fair can you please just recreate the PR to target the dev branch instead of master?

**Comment by PiZZAD0X** [comment]
> > > So how come start in vehicle is removed?
> > > Also this should target the dev branch please.
> > 
> > 
> > I looked at the module a few days ago and noticed no settings file for it so I asked pizza and he said it was a redundant module.
> 
> OK fair can you please just recreate the PR to target the dev branch instead of master?

it's not in dev branch, it is only left in master because the dev to master merge didn't delete the extra files

**Comment by PiZZAD0X** [comment]
Just needs the setting.sqf for the marker control updated so it has both Add and AddLayer examples

**Comment by TheLocalPub** [comment]
I done this today but just need to add it to the fork.

**Comment by PiZZAD0X** [comment]
> I done this today but just need to add it to the fork.

next time make a branch for the feature/bugfix on this fork so we can add to it

**Comment by PiZZAD0X** [comment]
for future reference once you "commit" to a branch that is already in a PR, the PR will automatically update. You can see the total changed files and commits included in the PR.

**Review by PiZZAD0X — APPROVED** [review]

**Pull request merged**